langwatch/src/pages/api/optimization/[...params].ts

```diff
@@ -1,29 +1,18 @@
 import { type NextApiRequest, type NextApiResponse } from "next";
-import { prisma } from "../../../server/db"; // Adjust the import based on your setup
-import { getProjectModelProviders } from "~/server/api/routers/modelProviders";
-import { type Workflow } from "~/optimization_studio/types/dsl";
-import { nanoid } from "nanoid";
-
-import type { StudioClientEvent } from "~/optimization_studio/types/events";
-
-import { type Edge, type Node } from "@xyflow/react";
-import { type MaybeStoredModelProvider } from "~/server/modelProviders/registry";
-import { addEnvs } from "~/optimization_studio/server/addEnvs";
-import { checkIsEvaluator } from "~/optimization_studio/utils/nodeUtils";
+import { executeWorkflowEvaluation } from "~/utils/executeEvalWorkflow";
```

Comment:
hmmm no I wouldn't put it in utils, please avoid putting things in utils unless they really are, well, utils. But this is intrinsically something of langwatch domain (utils is more like something that could be used by any app at all, like stringCasing, date functions, etc)

maybe create a server/optimization folder?